src/components/ToDoForm.vue

```diff
@@ -1,23 +1,127 @@
 <template>
-    <button @click.prevent="handleClick">Add</button
+  <div class="relative z-0 mt-8 h-fit w-full rounded-2xl border-2 border-black bg-transparent">
+    <div
+      v-if="isDropdownShowing || isDeletePopupVisible"
+      class="absolute h-full w-full bg-black opacity-50 phone:hidden"
+    ></div>
+    <div class="flex items-center justify-between">
+      <input
+        type="text"
+        class="ml-6 mt-4 flex h-12 w-8/12 items-center justify-center indent-0.5 font-title text-4xl font-semibold leading-10 text-black phone:ml-4 phone:h-6 phone:w-48 phone:text-lg"
+        :placeholder="updatedToDo.title"
+        v-model="updatedToDo.title"
+      />
+      <ToDoPriority
+        :priority="todo.priority"
+        @updateNewPriority="updateNewPriority"
+        @getFadeoutClass="setDropdownVisibility"
+      />
+    </div>
+    <div>
+      <input
+        v-model="toDoDate"
+        type="date"
+        name="dueDate"
+        id="dueDate"
+        min="2023-08-20"
```

Comment:
This should be dynamic, what happens if you open the app in October? Can you select 20th of August?